bundles/net.tourbook.common/src/net/tourbook/common/Hack.java

```diff
@@ -23,62 +23,50 @@
 
 public class Hack {
 
-	/**
-	 * Use Java Reflection to call protected Table#setItemHeight(int).
-	 * 
-	 * @param table
-	 *            Tree on which to adjust item height.
-	 * @param itemHeight
-	 *            height in pixels to allow for each item.
-	 */
-	public static void setTableItemHeight(final Table table, final int itemHeight) {
+   /**
+    * Use Java Reflection to call protected Table#setItemHeight(int).
+    *
+    * @param table
+    *           Tree on which to adjust item height.
+    * @param itemHeight
+    *           height in pixels to allow for each item.
+    */
+   public static void setTableItemHeight(final Table table, final int itemHeight) {
 
-		try {
+      try {
 
-			// Search class hierarchy for protected setter method
-			final Method setter = Table.class.getDeclaredMethod("setItemHeight", int.class);//$NON-NLS-1$
+         // Search class hierarchy for protected setter method
+         final Method setter = Table.class.getDeclaredMethod("setItemHeight", int.class);//$NON-NLS-1$
 
-			setter.setAccessible(true);
-			setter.invoke(table, Integer.valueOf(itemHeight));
+         setter.setAccessible(true);
+         setter.invoke(table, Integer.valueOf(itemHeight));
 
-		} catch (final NoSuchMethodException e) {
-			// ignore
-		} catch (final IllegalArgumentException e) {
-			// ignore
-		} catch (final IllegalAccessException e) {
-			// ignore
-		} catch (final InvocationTargetException e) {
-			// ignore
-		}
-	}
+      } catch (final NoSuchMethodException | IllegalArgumentException | IllegalAccessException | InvocationTargetException e) {

```

Comment:
@wolfgang-ch That was a nice feature introduced in Java 7 that allows to catch multiple exceptions in 1 catch only
[Source](https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html)